Resources/doc/getting_started/installation.rst

```diff
@@ -12,7 +12,7 @@ following command to download the latest stable version of this bundle:
 
 .. code-block:: bash
 
-    $ composer require sonata-project/admin-bundle "2.3.*"
+    $ composer require sonata-project/admin-bundle "3.0.*"
```

Comment:
This is not a good idea!

Symfony docs are using fixed versions too, to be sure that the docs are working with the examples

ping @javiereguiluz